protocol/avdl/chat1/local.avdl

```diff
@@ -1063,6 +1063,13 @@ protocol local {
   }
   GetTLFConversationsLocalRes getTLFConversationsLocal(string tlfName, TopicType topicType, ConversationMembersType membersType);
 
+  record GetChannelMembershipsLocalRes {
+    array<ChannelNameMention> channels;
+    boolean offline;
+    array<RateLimit> rateLimits;
+  }
+  GetChannelMembershipsLocalRes getChannelMembershipsLocal(string tlfName, keybase1.UID uid, TopicType topicType, ConversationMembersType membersType);
```

Comment:
Can simplify this to just be implied to be members type === TEAM, and topicType === CHAT

Reply:
Just have this also take a team ID, so you can further simplify (JS has these ready to go)